site/gr/index.html

```diff
@@ -0,0 +1,672 @@
+                    μέσω εκλεπτυσμένων APIs, άμεσα διαθέσιμων στα δικά σας collections.
+                </p>
+            </div>
+            <div class="col-lg-pull-5 col-lg-7 col-sm-pull-6  col-sm-6">
+                <ul class="nav nav-tabs">
+                    <li class="active"><a href="#method" data-toggle="tab">Eclipse Collections με Method Ref</a></li>
+                    <li><a href="#ec" data-toggle="tab">Eclipse Collections με Lambda expression</a></li>
+                    <li><a href="#streams" data-toggle="tab">Java 8 Streams</a></li>
+                </ul>
+                <div id="ec-vs-streams" class="tab-content">
+                    <div class="tab-pane fade in active" id="method">
+                            <pre  class="prettyprint lang-java">
+boolean anyPeopleHaveCats =
+  this.people
+    .anySatisfyWith(Person::hasPet, PetType.CAT);
+
+int countPeopleWithCats =
+  this.people
+    .countWith(Person::hasPet, PetType.CAT);
+
+MutableList&lt;Person&gt; peopleWithCats =
+  this.people
+    .selectWith(Person::hasPet, PetType.CAT)</pre>
+                    </div>
+                    <div class="tab-pane fade" id="ec">
+                            <pre  class="prettyprint lang-java">
+boolean anyPeopleHaveCats =
+  this.people
+    .anySatisfy(person -> person.hasPet(PetType.CAT));
+
+int countPeopleWithCats =
+  this.people
+    .count(person -> person.hasPet(PetType.CAT));
+
+MutableList&lt;Person&gt; peopleWithCats =
+  this.people
+    .select(person -> person.hasPet(PetType.CAT));</pre>
+                    </div>
+                    <div class="tab-pane fade" id="streams">
+                            <pre  class="prettyprint lang-java">
+boolean anyPeopleHaveCats =
+  this.people
+    .stream()
+    .anyMatch(person -> person.hasPet(PetType.CAT));
+
+long countPeopleWithCats =
+  this.people
+    .stream()
+    .filter(person -> person.hasPet(PetType.CAT))
+    .count();
+
+List&lt;Person&gt; peopleWithCats =
+  this.people
+    .stream()
+    .filter(person -> person.hasPet(PetType.CAT))
+    .collect(Collectors.toList());</pre>
+                    </div>
+                </div>
+                <!-- /.tab-content -->
+            </div>
+            <!-- /.col -->
+        </div>
+        <!-- /.row -->
+
+    </div>
+    <!-- /.container -->
+
+</div>
+<!-- Rich and fluent API -->
+
+<div class="content-section-a">
+
+    <div class="container-fluid">
+
+        <div class="row">
+            <div class="col-lg-5 col-sm-6">
+                <hr class="section-heading-spacer">
+                <div class="clearfix"></div>
+                <h2 class="section-heading">Πολλά container types συμπεριλαμβανομένων μεταξύ άλλων των <br>immutable collections, primitive collections, bimaps, multimaps και bags</h2>
+            </div>
+                <p class="lead">Μπορείτε να χρησιμοποιήσετε διάφορα είδη container, αξιοποιώντας εύχρηστα factory methods για αρχικοποίηση.</p>
```

Comment:
"αρχικοποίηση" may match "initialization" more than "instantiation". Would the following be a good alternative?

```suggestion
                <p class="lead">Μπορείτε να χρησιμοποιήσετε διάφορα είδη container, αξιοποιώντας εύχρηστα factory methods για δημιουργία αντικειμένων.</p>
```

Reply:
Yes! I like this one better, too